node/plan.sh

```diff
@@ -23,6 +23,16 @@ do_prepare() {
   fix_interpreter configure core/coreutils bin/env
 }
 
+do_build() {
+  ./configure \
+    --prefix "${pkg_prefix}" \
+    --dest-cpu "x64" \
+    --dest-os "linux" \
+    --fully-static
```

Comment:
Are we sure we want this flag? https://github.com/nodejs/node/wiki/statically-linked-executable has some caveats that make me think we might at least want to test with some modules that could be affected by its implications.

Will building fully static make it so there's no glibc dep either? The benefits of static linking might be minimal in a Habitat context since we have pre-determined library dependencies.